src/pages/ScoreReport.vue

```diff
@@ -802,6 +840,14 @@ const runsByTaskId = computed(() => {
   });
 });
 
+const sortedTaskIds = computed(() => {
+  const taskIdsInOrder = ['pa', 'swr', 'sre', 'letter', 'cva', 'fluency', 'mep', 'morph', 'multichoice', 'vocab'];
```

Comment:
This list shouldn't be needed, I recommend utilizing `taskDisplayNames` imported from `helpers/report.js`. For each task there's a taskId, display name, and ordering number. The reason I recommend this is so all of our ordering is in once place and consistent.

Reply:
Good recommendation! I will make the change.